module_1/gradle/by.epam.module1.gradle.utils/src/main/java/StringUtils.java

```diff
@@ -0,0 +1,7 @@
+import static org.apache.commons.lang3.StringUtils.isNumeric;
+
+public class StringUtils {
+    public static boolean isPositiveNumber (String str) {
+        return isNumeric(str);
```

Comment:
isPositiveNumber method mane sounds like you should check if the number is positive. isNumeric doesn't do this check